README.md

```diff
@@ -102,7 +102,7 @@ To override these settings, import `py42.sdk.settings` and override values as ne
 
```

Comment:
Looks like on line 100, still is using `py42.sdk.settings`

Reply:
good catch